BonusCal.py

```diff
@@ -2,12 +2,11 @@
 
 sal= int(input("Enter your salary in thousands: "))
 
-bonus=(0.15*sal)
+# bonus modified with 25%
+# modified date 11dec at 11.00am
+
+bonus=(0.25*sal)
 if exp>=2 and sal>=25000:
     print(f"Your bonus amount is Rs.{bonus}")
 else:
     print(f"Sorry, you are not eligible for bonus")
-
```

Comment:
why these lines are removed?

Reply:
These lines are added by me in last PR. and these line have no use.